src/customizing-workflow/tuning-component-settings.md

```diff
@@ -0,0 +1,46 @@
+# Tuning Component Settings
+
+Helm Charts are a set of Kubernetes manifests that reflect best practices to deploy an application
+or service. Helm is heavily influenced by [Homebrew](http://brew.sh/), including the
+[formula model](https://github.com/Homebrew/homebrew-core). A Helm chart is to Helm as a Formula
+is to Homebrew.
+
+When you run `helm fetch deis/workflow-beta1`, you can customize the chart with
+`helm edit workflow-beta1`. To customize the respective component, edit
```

Comment:
Will do. It was expected that this would be merged before we shipped beta2. Thanks for finding this!